k6/run-k6.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# k6 디렉터리로 이동
+workdir=$(dirname $0)
+cd $workdir
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)